practice-app/app.py

```diff
@@ -1,6 +1,7 @@
 from flask import Flask, jsonify
 import datetime
 from database import mongo
+from .errorHandlers import errorHandlers_bp
```

Comment:
```suggestion
from errorHandlers import errorHandlers_bp
```
This should also work. Relative imports creates some problems when unittesting.